### PR TITLE
Tentative update to include a term role test when calculating the accessible name for name-from-content elements

### PR DESCRIPTION
**Pull request by giacomo-petri**
Demo

### CODE REVIEW

File: accname/name/comp_name_from_content.html [hunk 1]
```diff
@@ -258,7 +258,39 @@ <h1 data-expectedlabel="Call us" data-testname="heading name from content with t
...
+<!-- Giacomo Petri first test, be patient if there is any mistake :D -->
+<h1>Name from content containing element with implicit term role</h1>
+<p>This test aims to verify the computed acc name of elements (buttons, headings, links) containing a term role.</p>
+<p>This test covers:</p>
+<ul>
+  <li>Elements with implicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with implicit button, heading, link roles containing element with explicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with explicit term role</li>
+</ul>
```
Comment (cookiecrook) [inline accname/name/comp_name_from_content.html]: This level of detail is not necessary, since it's covered already in the comments and test names below.
```suggestion
```

File: accname/name/comp_name_from_content.html [hunk 2]
```diff
@@ -258,7 +258,39 @@ <h1 data-expectedlabel="Call us" data-testname="heading name from content with t
...
+<p>Tests the <a href="https://www.w3.org/TR/accname-1.2/#comp_name_from_content">Computation steps</a> (specifically point F) from the Accessible Name and Description Computation 1.2 specifications for that specific instance.</p>
```
Comment (cookiecrook) [inline accname/name/comp_name_from_content.html]: This can be an HTML comment rather than a rendered paragraph.

It should also specifically mention the `term` role since that the edge case this new set of tests is covering, right?

Comment (cookiecrook) [reply accname/name/comp_name_from_content.html]: Also never reference the point by the outline alphanumeric, since that changes sometimes... so instead of (F)(iv) you'd reference the named anchor _**Name from Each Child**_. (#comp_name_from_content_for_each_child)

https://w3c.github.io/accname/#comp_name_from_content_for_each_child

Comment (cookiecrook) [reply accname/name/comp_name_from_content.html]:
```suggestion
<!-- Tests `Name from Each Child` edge case failures with an embedded `term` as a child element role. -->
<!-- https://w3c.github.io/accname/#comp_name_from_content_for_each_child -->
```

**Review by cookiecrook — CHANGES_REQUESTED** [review]
This needs to be a PR on the main WPT repo, because that integrates the test results checks... You can commit to your own fork (I do too) but the PR should point to WPT to get those integration checks, so we can verify the test results before merging.

Thanks for working on this!

**Comment by cookiecrook** [comment]
Related issue in the [Interop Accessibility](https://github.com/web-platform-tests/interop-accessibility/) repo.
- https://github.com/web-platform-tests/interop-accessibility/issues/157
